### DISCUSSION TITLE
How do I activate Airplay 1 compatibility mode?

### DISCUSSION THREAD

**Discussion by theblond** [mikebrady/shairport-sync#1289] [Q&A] [answered]
I read this about the Airplay 2 implementation:
[https://github.com/mikebrady/shairport-sync/blob/development/AIRPLAY2.md](https://github.com/mikebrady/shairport-sync/blob/development/AIRPLAY2.md)

How do I activate the mentioned Airplay 1 compatibility mode? I use an HifiBerryOS which uses Shairport for Airplay 2. But I cannot playback music in iTunes on my Windows machine for known reasons. Is there a line I have to write into the shairport-sync.conf file or how do I do that?

**Comment by charlesomer** [comment, marked as answer]
Hi, unfortunately there are problems when sending audio from iTunes on Windows at the moment with the development version. I suggest you just use the main branch for now or the normal docker images :)

**Reply by theblond** [reply]
ok thanks!
